frontend/src/components/common/__snapshots__/ActionsNotifier.stories.storyshot

```diff
@@ -11,14 +11,14 @@ exports[`Storyshots ActionsNotifier Some 1`] = `
   id="root"
 >
   <div
-    class="makeStyles-root makeStyles-bottom makeStyles-left makeStyles-bottom makeStyles-left makeStyles-reverseColumns"
+    class="makeStyles-bottom makeStyles-left makeStyles-root"
```

Comment:
Seems that some of the styles were duplicated before, and are only in there once now -- good.

However it seems `makeStyles-reverseColumns` was removed... so this needs checking.

Reply:
This is an internal to notistack-snackbar style change. Nothing to worry about.